include/rocksdb/utilities/transaction_db.h

```diff
@@ -396,6 +396,9 @@ struct TransactionOptions {
   // comment for `commit_bypass_memtable` for more optimization detail.
   uint32_t large_txn_commit_optimize_threshold =
       std::numeric_limits<uint32_t>::max();
+
+  uint64_t large_txn_commit_optimize_byte_threshold =
```

Comment:
nit: Would it help if these type of documentation can be added for this set of options:
1) If any transaction's commit should bypass memtable, set `commit_bypass_memtable` to true.
2) If only when a transaction has n or n+ operations should bypass memtable, set `commit_bypass_memtable` to false and large_txn_commit_optimize_threshold to n and large_txn_commit_optimize_byte_threshold to max.
3) If only when a transaction has with n or n+ bytes should bypass memtable. set ...

Reply:
Added comment for this.